src/utils/constants.js

```diff
@@ -1,3 +1,5 @@
-const AUTH_STORAGE_KEY = 'wa_cert_authenticated';
-
-export { AUTH_STORAGE_KEY };
+export default {
+  AUTH_STORAGE_KEY: 'wa_cert_authenticated',
+  API_URL: 'https://www.googleapis.com/youtube/v3/',
+  API_KEY: 'AIzaSyA2RrmXuKEobD96oL8gnqZz1Mh52iBUNd8',
```

Comment:
I suggest never uploading secrets to the Github repo, so I suggest adding the `API_KEY` into the `.env` file. Also, I recommend creating a .env.example, track the file, and put there the necessary variables
.env
```
REACT_APP_API_KEY=<value_of_the_key>
```
src/utils/constants.js
```
...
export default {
  ...,
  API_KEY: process.env.REACT_APP_API_KEY,
}
```
.env.sample
```
REACT_APP_API_KEY=
```
[Official create react documentation](https://create-react-app.dev/docs/adding-custom-environment-variables/)